datastream-server/src/main/java/com/linkedin/datastream/server/EventProducer.java

```diff
@@ -60,6 +60,7 @@ public class EventProducer implements DatastreamEventProducer {
   static final String EVENTS_SEND_LATENCY_MS_STRING = "eventsSendLatencyMs";
   static final String THROUGHPUT_VIOLATING_EVENTS_LATENCY_MS_STRING = "throughputViolatingEventsLatencyMs";
   static final String THROUGHPUT_VIOLATING_EVENTS_SEND_LATENCY_MS_STRING = "throughputViolatingEventsSendLatencyMs";
+  static final String NOOP_TRANSPORT_PROVIDER_NAME = "NoOp";
```

Comment:
Can this string constant be reused?

Reply:
Done. Moved the constant to `NoOpTransportProvider` class.